src/Modules/OrchardCore.Commerce/Views/Payment/Success.cshtml

```diff
@@ -0,0 +1,16 @@
+<div class="row pt-md-5">
+    <div class="col-6 mx-auto">
+        <div class="panel panel-primary">
+            <div class="panel-heading">
+                <h4 class="text-success text-center">@T["Thank you for your purchase!"]</h4>
+            </div>
+            <div class="panel-body">
+                <p>
+                    <b>@T["The receipt will be sent to your email address!"] </b>
```

Comment:
Maybe I messed something up, but I tried this out and it didn't send an email to either Papercut or my actual address. Is this supposed to work or just a placeholder?

Reply:
No, that's just a placeholder.

Reply:
Then please add something like "once we have processed your order" to the end, so people don't get the wrong idea.